frontend/src/components/cronjob/Details.tsx

```diff
@@ -31,11 +60,74 @@ export default function CronJobDetails() {
 
   const ownedJobs = filterOwnedJobs(jobs);
 
+  function handleCron(suspend: boolean) {
+    const suspendingOrStarting = suspend ? 'Suspend' : 'Resum';
+    cronJob.spec.suspend = suspend;
+    // when we apply the cronjob, we need to remove the resourceVersion because a new resourceVersion is generated
+    // when the cronjob is applied
+    delete cronJob.metadata.resourceVersion;
+    setIsCheckingCronSuspendStatus(true);
+    dispatch(
+      clusterAction(() => applyFunc(cronJob.jsonData), {
+        startMessage: t(`resource|${suspendingOrStarting}ing CronJob {{ newItemName }}…`, {
```

Comment:
The translation function works by matching the strings after the `|` separator with the catalog of translations we have.
The catalog is populated by scanning (parsing) the code and extracting the values you have added to the function.

This means the string in `t()` needs to be a plain string and not a literal. So computing strings within that function will not work with translations. Please have different strings for different cases if you need to branch out here.

Reply:
fixed

Reply:
I squashed a change to remove the back ticks (this causes a warning) + remove the capitalization of verbs (Supended -> suspended)